macOS-HS-libvirt.xml

```diff
@@ -66,18 +61,16 @@
     <feature policy='require' name='xsave'/>
     <feature policy='require' name='avx'/>
     <feature policy='require' name='xsaveopt'/>
-    <feature policy='require' name='avx2'/>
```

Comment:
The default should probably support modern CPUs fully. Folks having older CPUs should change this setting themselves.

Reply:
okay I readded the line